### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by MoLow**
fixes: https://github.com/nodejs/tap2junit/issues/33

### CODE REVIEW

File: .github/workflows/lint_python.yml [hunk 1]
```diff
@@ -14,16 +14,9 @@ jobs:
       - run: codespell --skip="*.tap"  # --ignore-words-list=""
```
Comment (cclauss) [inline .github/workflows/lint_python.yml]: Please change line 12 to `- run: bandit --recursive --skip=B101,B604,B607 .` to placate `bandit`.

File: test/_prepare_outputs.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import os
+
+for file in glob.glob('test/fixtures/*.tap'):
+    name = os.path.basename(file).replace('.tap', '')
+    try: 
+        run = subprocess.run(["python", "-m", "tap2junit", "-i", file, "-o", f"test/output/{name}.xml"])
+        run.check_returncode()
+    except subprocess.CalledProcessError as e:
+        print(f"failed running test {name}")
+
```
Comment (cclauss) [inline test/_prepare_outputs.py]: 🚫 means that GitHub wants files to end with one and only one blank line.

**Comment by MoLow** [comment]
I copied the change from https://github.com/nodejs/tap2junit/pull/34 to make sure tests pass

File: test/_common.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import platform
+
+replacements = dict([(platform.node(), "{HOSTNAME}")])
+
+
+def normalize_output(str):
+    for replacement in replacements:
+        str = str.replace(replacement, replacements[replacement])
+    return str
```
Comment (cclauss) [inline test/_common.py:9]: `str` is a Python type so we should not shadow it.
```suggestion
def normalize_output(s):
    for replacement in replacements:
        s = s.replace(replacement, replacements[replacement])
    return s
```

**Review by cclauss — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Pull request merged**